README.md

```diff
@@ -59,6 +59,13 @@ $ go install github.com/google/osv-scanner/cmd/osv-scanner@v1
 
 This requires Go 1.18+ to be installed.
 
+### Releasing
```

Comment:
I think it would be better to title this "Building", and frame it from that POV - so start with explaining that you can build from source using `goreleaser build --single-target`, and then mention that version builds can be reproduced by checking out the specific tag and doing a build.

Is probably also worth linking to the [`goreleaser build` docs](https://goreleaser.com/cmd/goreleaser_build/)